portfolio/src/main/webapp/script.js

```diff
@@ -32,3 +32,15 @@ function showFact() {
     const factContainer = document.getElementById('fact-container');
     factContainer.innerText = fact;
 }
+
+
+async function getMessages() {
+    console.log("Getting the messages...");
```

Comment:
If this is for debugging purpose, let's remove it before merging.

Reply:
Good to know!